tuning_docs/tensile_tuning.tex

```diff
@@ -579,61 +637,97 @@ \subsubsection{The Tensile Logic File}
+rocBLAS/build/release/client/staging$ ./rocBLAS-bench -f gemm -r f32_r --transposeA N --transposeB T -m 1001 -n 1536 -k 64 --alpha 1.0 --lda 1001 --ldb 1536 --beta 0.0 --ldc 1001 -i 10
+rocBLAS/build/release/client/staging$ ./rocBLAS-bench -f gemm_ex --transposeA N --transposeB T -m 1024 -n 1024-k 1024 --a_type h --lda 2048 --b_type h --ldb 2048 --beta 1 --c_type s --ldc 2048 --d_type s --ldd 2048 --compute_type s -v 1 --iters 10 --cold_iters 2
 \end{lstlisting}
 
 \noindent
-Here, the i option is the number of iterations of the gemm call. Because there is a latency in loading the Tensile library which rocBLAS links to, for testing, it is recommended to use rocblas-bench with the batch call. The batch call takes a yaml file with a list of problem sizes. An example yaml file contents is shown below.
+Here, the i/iters option is the number of iterations of the gemm call. See the help output for other flags. Because there is a latency in loading the Tensile library which rocBLAS links to, for testing, it is recommended to use rocBLAS-bench with the batch call. The batch call takes a yaml file with a list of problem sizes. An example yaml file contents is shown below (problem-size.yaml).
 
 \begin{minted}[
 gobble=0,breaklines,
 frame=single,
 linenos
 ]{yaml}
 
-- { rocblas_function: "rocblas_sgemm", transA: 'N', transB: 'N', M: 1024, N: 1264, K: 345, lda: 1024, ldb: 345, ldc: 1024, cold_iters: 2, iters: 10 }
-- { rocblas_function: "rocblas_sgemm", transA: 'N', transB: 'N', M: 1025, N: 1064, K: 148, lda: 1025, ldb: 148, ldc: 1025, cold_iters: 2, iters: 10 }
-- { rocblas_function: "rocblas_sgemm", transA: 'N', transB: 'N', M: 2450, N: 512, K: 1023, lda: 2450, ldb: 1023, ldc: 2450, cold_iters: 2, iters: 10 }
-- { rocblas_function: "rocblas_sgemm", transA: 'N', transB: 'N', M: 2345, N: 2222, K: 345, lda: 2345, ldb: 345, ldc: 2345, cold_iters: 2, iters: 10 }
+- { rocBLAS_function: "rocBLAS_sgemm", transA: 'N', transB: 'N', M: 1024, N: 1264, K: 345, lda: 1024, ldb: 345, ldc: 1024, cold_iters: 2, iters: 10 }
+- { rocBLAS_function: "rocBLAS_sgemm", transA: 'N', transB: 'N', M: 1025, N: 1064, K: 148, lda: 1025, ldb: 148, ldc: 1025, cold_iters: 2, iters: 10 }
+- { rocBLAS_function: "rocBLAS_gemm_ex", transA: "N", transB: "T", a_type: f16_r, b_type: f16_r, c_type: f32_r, d_type: f32_r, compute_type: f32_r, M:    4096, N:    4096, K:    1024, lda:   8192, ldb:   8192, ldc:   8192,  ldd:   8192, cold_iters: 2, iters: 10, beta: 1 }
+- {rocBLAS_function: rocBLAS_gemm_ex, transA: N, transB: T, a_type: f16_r, b_type: f16_r, c_type: f32_r, d_type: f32_r, compute_type: f32_r, M: 1024, N: 1024, K: 1024, lda: 8192, ldb: 8192, ldc: 8192, ldd: 8192, alpha: 1, beta: 1, iters: 1, cold_iters: 0, norm_check: 1}
 
 \end{minted}
 
 \noindent
-This can be executed in rocblas-bench as follows:
+This can be executed in rocBLAS-bench as follows:
 
 \begin{lstlisting}[language=bash]
-./rocblas-bench --yaml problem-sizes.yaml
+./rocBLAS-bench --yaml problem-sizes.yaml
```

Comment:
This definitely _is_ case sensitive. I'm guessing a find and replace got out of hand?